norns/study-1.md

```diff
@@ -527,6 +527,20 @@ function key(n,z)
 
 **Be sure to keep an eye on the REPL for errors.**
 
+### deeper debugging
+
+A lot of the time we'll see errors occur in maiden. But sometimes that won't be enough. For example, maiden only allows you to scroll back through messages so far, and occasionally (for exmaple if you're [developing a mod](https://monome.org/docs/norns/community-scripts/#mods)) maiden will hide any errors.
```

Comment:
rad tytyty!

correcting some typos:

> We'll see errors from the running script print to maiden's REPL. But since maiden only allows you to scroll back through a limited history and occasionally will suppress errors (for example if you're [developing a mod](/docs/norns/community-scripts/#mods)), that sometimes won't be enough.